test/cors.test.js

```diff
@@ -175,11 +175,11 @@ test('Should support dynamic config (callback)', t => {
     t.equal(res.payload, 'ok')
     t.match(res.headers, {
       'access-control-allow-origin': 'example.com',
-      vary: 'Origin',
```

Comment:
the test case says `Should support dynamic config (callback)`

And should Vary on origin.

Reply:
and now?

Reply:
https://github.com/fastify/fastify-cors/issues/291